.github/workflows/build_and_publish.yaml

```diff
@@ -0,0 +1,100 @@
+name: Tag repo; build and publish assets
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  tag-new-version:
+    runs-on: ubuntu-latest
+    outputs:
+      tag: ${{ steps.tag.outputs.new_tag }}
+      version: ${{ steps.tag.outputs.new_version }}
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 0
+      - name: Bump version and push tag
+        id: tag
+        uses: mathieudutour/github-tag-action@v4.5
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          default_bump: false
+          release_branches: main
+
+  build-and-publish-package:
+    runs-on: ubuntu-latest
+
+    name: Build and publish PyPI package
+    needs: tag-new-version
+    if: needs.tag-new-version.outputs.tag
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v1
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+
+    - name: Install wheel package
+      run: |
+        pip install wheel
+
+    - name: Generate correct value for VERSION file
+      run: |
+        echo ${{ needs.tag-new-version.outputs.tag }} > opensafely/VERSION
```

Comment:
should `opensafely/VERSION` be `osgithub/VERSION`?

Reply:
Fixed now